### DISCUSSION TITLE
Macro View Empty yet can execute macros by right clicking measure

### DISCUSSION THREAD

**Discussion by lstee014** [TabularEditor/TabularEditor3#707] [🙋♀️TE3 Q&A]
My measure view is completely blank despite writing and saving several measures.  I am able to execute the macros by right clicking a measure (at that time I see all my macros).  The issue is that I am unable to delete or edit any of them. Any idea what the issue could be?

**Comment by otykier** [comment]
Which version of TE3 are you using?

One thing you could try, is to reset the window layout (**Window > Default layout**), then restart the application.

**Reply by lstee014** [reply]
Hello -  Thank you for the quick reply.  I am using 3.3.5.  I have tried the reset layout, even uninstalled and reinstalled the program today, but unfortunately it did not help.

**Reply by lstee014** [reply]
![TE3 Macro view](https://user-images.githubusercontent.com/67977985/200850860-b71bd5b5-0d3f-401d-9182-96b183fb4cf9.PNG)

**Reply by otykier** [reply]
Thanks for sharing these additional details. We have identified a bug, which will be fixed in the next update, which will be available in a day or two. Will keep you posted!

In the mean time, if you need to edit specific macros, you can find the definition of all your macros in the `%LocalAppData%\TabularEditor3\MacroActions.json` file.

**Reply by lstee014** [reply]
Thank you!!!!

**Reply by otykier** [reply]
Fixed in [3.3.6](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.3.6)!

**Reply by lstee014** [reply]
Awesome!
